docs/_documentations/mdt-eclipse-firstproject.md

```diff
@@ -1,46 +1,48 @@
 ---
 layout: docs
-title: "Codewind for Eclipse"
-description: "Codewind for Eclipse"
+title: "Creating your first Eclipse Codewind project"
+description: "Creating your first Eclipse Codewind project"
 keywords: introducing, introduction, overview, what is, tools, vscode, visual, studio, code, java, microprofile, spring, node, nodejs, node.js, javascript, Codewind for VS Code, tools, view, debug, integrate, open a shell session, toggle auto build, manually build, scope VS Code workspace, disable, enable, delete
 duration: 1 minute
 permalink: mdt-eclipse-firstproject
 type: document
 order: 2
 parent: mdt-eclipse-overview
 ---
-# Creating your first project
+# Creating your first Eclipse Codewind project
 <br/>
-To create your first project, right-click the **Local [Running]** item and select **Create New Project...**
+To create your first project:
+
+1\. Right-click the **Local [Running]** item >  select **Create New Project...**.
```

Comment:
- Keep the "and" here instead of the `>`. The `>` is OK for if you have a long string of things to click, such as, "Go to `Program>Files>Product_Page` and click the file you want to edit."
- Also delete the extra space before "select."